src/main/java/uk/gov/hmcts/reform/idam/health/indicator/HealthCheck.java

```diff
@@ -28,14 +28,15 @@ public Health health() {
             builder = Health.up();
         } else {
             builder = Health.down();
-            patients.stream().forEach(indicator -> {
-                        String details = indicator.getDetails();
-                        if (details != null) {
-                            builder.withDetail(indicator.getClass().getName(), details);
-                        }});
+            patients.forEach(indicator -> {
+                String details = indicator.getDetails();
+                if (details != null) {
+                    builder.withDetail(indicator.getClass().getName(), details);
+                }});
         }
         return builder
-                .withDetail("v", defaultIfEmpty(getClass().getPackage().getImplementationVersion(), "dev"))
+                //TODO Put the version in the jar manifest.
+                .withDetail("v", defaultIfEmpty(getClass().getPackage().getImplementationVersion(), "2.0.3"))
                 .build();
```

Comment:
![image](https://user-images.githubusercontent.com/47384516/74816133-72372000-52f2-11ea-97c7-4ded033c469f.png)

Reply:
I've seen this but it feels like a workaround. I think it is better to put the right version in the MANIFEST.